### PR TITLE
Edrd mar release24 defect fixing branch

### PR DESCRIPTION
**Pull request by deepakmulamalla**
Defects fixing Changes & Adding ShareSets

### CODE REVIEW

**Review by Rajashreeacn — APPROVED** [review]

**Review by cgijeffolsen — APPROVED** [review]
These changes are all appropriate for EDRD release. Approved. @deepakmulamalla 
cc @NataliaNikishina

**Comment by deepakmulamalla** [comment]
> These changes are all appropriate for EDRD release. Approved. @deepakmulamalla cc @NataliaNikishina

Thanks @cgijeffolsen for approving PR.

**Comment by deepakmulamalla** [comment]
@cgijeffolsen, I'm reverting the Case level access for EDRD Roles

EDRD_Staff & EDRD_Admin as Case access level below organization default as scratch org definition file as case default as readWriteTransfer

File: force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls [hunk 1]
```diff
@@ -90,7 +90,8 @@ public with sharing class ESA_cls_accountTriggerHandler {
             for(Case caseObj: casesToUpdate){
                 if(caseObj.Account.Patient_is_Deceased__pc){
                     caseObj.status = 'Cancelled';
-                    caseObj.EDRD_Case_closure_reasons__c = 'Patient Deceased';
+                    caseObj.EDRD_Case_Closure_Reasons__c = 'Patient Deceased';
```
Comment (yashrohan-cgi) [inline force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls:93]: Can this be added to label ?

Comment (deepakmulamalla) [reply force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls:93]: @yashrohan-cgi, those are picklist values. Usually we add only Error Messages or Display messages to labels.

Comment (deepakmulamalla) [reply force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls:93]: The above one is the picklist value

Comment (deepakmulamalla) [reply force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls:93]: @yashrohan-cgi , any updates on this PR?

Comment (yashrohan-cgi) [reply force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls:93]: ideally any kind of string constants should be added to label . @deepakmulamalla

Comment (deepakmulamalla) [reply force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls:93]: @yashrohan-cgi, moved the String values to constants.

**Review by yashrohan-cgi — APPROVED** [review]

**Review by cgijeffolsen — APPROVED** [review]
Revisions as expected. Thank you @deepakmulamalla 
@NataliaNikishina

**Pull request merged**